src/main/kotlin/org/rust/lang/core/stubs/StubImplementations.kt

```diff
@@ -166,6 +166,7 @@ fun factory(name: String): RsStubElementType<*, *> = when (name) {
     "PATH_EXPR" -> RsExprStub.Type("PATH_EXPR", ::RsPathExprImpl)
     "RANGE_EXPR" -> RsExprStub.Type("RANGE_EXPR", ::RsRangeExprImpl)
     "RET_EXPR" -> RsExprStub.Type("RET_EXPR", ::RsRetExprImpl)
+    "YIELD_EXPR" -> RsExprStub.Type("RET_EXPR", ::RsRetExprImpl)
```

Comment:
I think we should increase stub version because if this change

Reply:
Fixed